dev-python/pydevd/pydevd-2.9.5.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
```

Comment:
How did you test it?

Reply:
I didn't, this was WIP for ipython. Descoped for now